pdfreader/codecs/decoder.py

```diff
@@ -14,7 +14,7 @@
 
 
 class PredefinedCmaps(object):
-    _cache = dict()
```

Comment:
I can see the benefit of these changes. dict() is slower than {}, and less pythonic.